packages/parser/tests/unit/schema/apigwv2.test.ts

```diff
@@ -98,6 +98,21 @@ describe('API Gateway HTTP (v2) Schemas', () => {
       // Assess
       expect(parsedEvent).toEqual(event);
     });
+
+    it('should parse the authorizer event with null identitySource', () => {
+      // Prepare
+      const event = getTestEvent({
+        eventsPath,
+        filename: 'authorizer-request',
+      });
+      event.identitySource = null;
```

Comment:
Since we have established that when not present `identitySource` is actually `null` as opposed to not being present at all, I think we should instead modify the test payload in `packages/parser/tests/events/apigw-rest/authorizer-request.json` and add the field there.

This way we can avoid adding an extra test and use the one at L88 which tests the same payload. With the field set to `.nullish()` both cases are covered by that case.

Reply:
`authorizer-request.json` has an array of strings as value, so we have two values we need to test, `null` and array of strings.

Reply:
You are right, I was looking at the REST folder.